### PR TITLE
Removing certificates.k8s.io/v1beta1 in favor of certificates.k8s.io/v1

### PR DESCRIPTION
**Pull request by Alevsk**
- CertificateSigningRequest is deprecated in v1.19+
- Kubernetes minimum requirement for Operator is  `v1.19`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ early development using MinIO for object storage.
 
 ## Prerequisites
 
-- MinIO requires Kubernetes version 1.17.0 or later.
+- MinIO requires Kubernetes version 1.19.0 or later.
```
Comment (ravindk89) [inline README.md:70]:
```suggestion
- Starting with Operator v4.0.0, MinIO requires Kubernetes version 1.19.0 or later. Previous versions of the Operator supported Kubernetes 1.17.0 or later. You must upgrade your Kubernetes cluster to 1.19.0 or later to use Operator v4.0.0+.
```

Comment (ravindk89) [reply README.md:70]: Only necessary if we want to be nice and extra prescriptive around this change.

File: README.md [hunk 2]
```diff
@@ -197,7 +197,7 @@ to temporarily forward traffic from the local host to the MinIO Tenant.
 
 MinIO supports expanding an existing MinIO Tenant onto additional hosts and storage.
 
-- MinIO requires Kubernetes version 1.17.0 or later.
+- MinIO requires Kubernetes version 1.19.0 or later.
```
Comment (ravindk89) [inline README.md:200]:
```suggestion
- Starting with Operator v4.0.0, MinIO requires Kubernetes version 1.19.0 or later. Previous versions of the Operator supported Kubernetes 1.17.0 or later. You must upgrade your Kubernetes cluster to 1.19.0 or later to use Operator v4.0.0+.
```

**Review by ravindk89 — APPROVED** [review]
No objections from a docs perspective.

One thought - this requires users to upgrade their K8s clusters to 1.19 even if they previously had a working tenant. Not sure if this needs a larger callout, but based on internal convos 1.18 is old enough where it seems like requiring a cluster upgrade is reasonable.

**Review by dvaldivia — APPROVED** [review]

**Pull request merged**